apps/token/src/routes/governance/components/propose/proposal-form-terms.spec.tsx

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from '@testing-library/react';
+import { ProposalFormTerms } from './proposal-form-terms';
+
+describe('Proposal Form Terms', () => {
+  it('should display error text', () => {
```

Comment:
chore (non-blocking): this is pretty thin, I'd like to see a test to validate that the docs URL is there adn the terms are rendered